R/save.R

```diff
@@ -1,6 +1,6 @@
 #' @rdname plot_carp
 #' @export
-saveviz <- function(x, ...) {
+saveviz <- function(x,frames_number,frame_rate, ...) {
```

Comment:
I think these should be arguments of the `saveviz.CARP` and `saveviz.CBASS` methods only - not the generic.